lib/functions/lintly.sh

```diff
@@ -40,19 +46,25 @@ function InvokeLintly() {
   LINTLY_FORMAT="${1}"
   LINTER_COMMAND_OUTPUT="${2}"
 
-  echo "----<<<<INVOKING Invokelintly>>>>----"
-  echo "FORMAT: ${LINTLY_FORMAT}"
-  echo "OUTPUT: ${LINTER_COMMAND_OUTPUT}"
-  echo ""
-  echo "DONE DISPLAYING ARGUMENTS"
+  debug "----<<<<INVOKING Invokelintly>>>>----"
+  debug "FORMAT: ${LINTLY_FORMAT}"
+  debug "OUTPUT: ${LINTER_COMMAND_OUTPUT}"
+  debug ""
+  debug "DONE DISPLAYING ARGUMENTS"
 
   # Lintly will comment on the PR
   echo "$LINTER_COMMAND_OUTPUT" | lintly --format="${LINTLY_FORMAT}"
 
-  echo "$?"
-  echo "^^ exit code ^^"
+  debug "$?"
+  debug "^^ exit code ^^"
 }
-
+################################################################################
+#### Function IsLintly #########################################################
+function IsLintly() {
+  [[ "${OUTPUT_MODE}" == lintly ]]
+}
```

Comment:
Is this named `IsLintly` to conform to the `IsTap` method?

I feel like maybe it could be named `OutputtingToLintly` or something.

Oh who knows. Naming is difficult.

Reply:
Yeah exactly that, it's to conform to the `IsTap` method. I don't like the naming either but I think it's still better to stay consistent.

Reply:
I'm going to keep it as-is for consistency, but let me know if you feel strongly about it.